.github/workflows/aws-docker-deploy.yml

```diff
@@ -36,6 +50,22 @@ jobs:
         id: login-ecr
         uses: aws-actions/amazon-ecr-login@v1
 
+      - name: Fill .env file
+        env:
+          AWS_USER_POOL_ID: ${{ inputs.aws-user-pool-id }}
+          AWS_WEB_CLIENT_ID: ${{ inputs.aws-user-pool-web-client-id }}
+          FRONTEND_CLIENT_API_URL: ${{ inputs.frontend-client-api-url }}
+          AWS_REGION: ${{ inputs.aws-region }}
+        run: |
+          touch ${{ inputs.client-environment-file-path }}
+          echo NEXT_PUBLIC_API_URL="FRONTEND_CLIENT_API_URL" >> ${{ inputs.client-environment-file-path }}
+          echo NEXT_PUBLIC_COGNITO_REGION="$AWS_REGION" >> ${{ inputs.client-environment-file-path }}
+          echo NEXT_PUBLIC_COGNITO_USER_POOL_ID="$AWS_USER_POOL_ID" >> ${{ inputs.client-environment-file-path }}
+          echo NEXT_PUBLIC_COGNITO_CLIENT_ID="$AWS_WEB_CLIENT_ID" >> ${{ inputs.client-environment-file-path }}
+          echo ".env created"
+          cat ${{ inputs.client-environment-file-path }}
+        shell: bash
```

Comment:
could be improved, maybe pass a whole file at once instead of params and write the file line per line?

Reply:
I find it a bit weird too to write the file line by line. Is there another way ? (maybe like having a template file and replacing variables maybe ?